pyradiosky/tests/test_skymodel.py

```diff
@@ -3,13 +3,15 @@
 # Licensed under the 3-clause BSD License
 
 import os
-import fileinput
 import re
+import fileinput
 
+import copy
```

Comment:
Super nitpicky: I think copy is part of the standard lib so should be in the top block of imports.